zdm/misc_functions.py

```diff
@@ -402,6 +402,8 @@ def make_dm_redshift(grid,savename,DMmax=1000,zmax=1,loc='upper left',Macquart=N
 	
 	plt.legend(loc=loc)
 	plt.savefig(savename)
+	plt.title("H0 " + str(H0))
+    plt.show()
```

Comment:
"plt.show" is infinitely annoying when not part of some option to the function - having plots come up on screen automatically drives me nut. Can you at least put the plt.show inside some kind of
if show:
     plt.show()
 and have "show=False" by default?

Reply:
Sure, I will make this change right away. Sorry for that! My IDE shows the plots in a different pane, so it wasn't a problem for me.